### PR TITLE
change urls https://eclipse.dev to  https://jetty.org

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
   - name: Jetty Security Reports
-    url: https://eclipse.dev/jetty/security_reports.php
+    url: https://jetty.org/security_reports.php
```
Comment (joakime) [inline .github/ISSUE_TEMPLATE/config.yml]: This URL doesn't exist.
We should create one and use it here.

File: .github/ISSUE_TEMPLATE/issue-template.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ labels: Bug
 **OS type/version**
 
 **Description**
-<!-- Do not report security issues here! See [Jetty Security Reports](https://eclipse.dev/jetty/security_reports.php) -->
+<!-- Do not report security issues here! See [Jetty Security Reports](https://jetty.org/security_reports.php) -->
```
Comment (joakime) [inline .github/ISSUE_TEMPLATE/issue-template.md]: This URL doesn't exist.
We should create one and use it here.

File: .github/ISSUE_TEMPLATE/release-template.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ This release process will produce releases:
 - [ ] Merge release branches back to main branches and delete release branches.
 - [ ] Verify release existence in Maven Central by triggering the Jenkins builds of CometD.
 - [ ] Update Jetty versions on the website ( follow instructions in [jetty-website](https://github.com/eclipse/jetty-website/blob/master/README.md) ).
-  + [ ] Update (or check) [Download](https://eclipse.dev/jetty/download.php) page is updated.
+  + [ ] Update (or check) [Download](https://jetty.org/download.php) page is updated.
```
Comment (joakime) [inline .github/ISSUE_TEMPLATE/release-template.md]: Wrong URL

File: jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -13,7 +13,7 @@
     <servlet-class>org.eclipse.jetty.ee10.proxy.ProxyServlet$Transparent</servlet-class>
     <init-param>
       <param-name>proxyTo</param-name>
-      <param-value>https://eclipse.dev/jetty/javadoc/jetty-12/index.html?overview-summary.html</param-value>
+      <param-value>https://javadoc.jetty.org/jetty-12/index.html</param-value>
```
Comment (joakime) [inline jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml]: Wrong URL

File: jetty-ee9/jetty-ee9-demos/jetty-ee9-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml [hunk 5]
```diff
@@ -13,7 +13,7 @@
     <servlet-class>org.eclipse.jetty.ee9.proxy.ProxyServlet$Transparent</servlet-class>
     <init-param>
       <param-name>proxyTo</param-name>
-      <param-value>https://eclipse.dev/jetty/javadoc/jetty-12/index.html?overview-summary.html</param-value>
+      <param-value>https://jetty.org/javadoc/jetty-12/index.html?overview-summary.html</param-value>
```
Comment (joakime) [inline jetty-ee9/jetty-ee9-demos/jetty-ee9-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml]: Wrong URL

**Review by joakime — CHANGES_REQUESTED** [review]
Got some URLs to fix.

File: jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml (hunk 4 above)
Comment (olamy) [reply jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml]: sure?

Comment (joakime) [reply jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml]: The lack of query section threw me on this one.
But it's correct.

File: jetty-ee9/jetty-ee9-demos/jetty-ee9-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml (hunk 5 above)
Comment (joakime) [reply jetty-ee9/jetty-ee9-demos/jetty-ee9-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml]: This one has both the hostname fault, and the query section fault.

Comment (joakime) [reply jetty-ee9/jetty-ee9-demos/jetty-ee9-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml]: Checked for other ones in `jetty-12.0.x`, but just found one that is a valid test case. (no need to change it)

``` shell
[12.0.x]$ git grep "jetty.org/.*?"
jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/FileIDTest.java:            Arguments.of(URI.create("https://jetty.org/docs.html?query=val#anchor"), "docs.html")
```

Comment (olamy) [reply jetty-ee9/jetty-ee9-demos/jetty-ee9-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml]: fixed

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (olamy) [reply .github/ISSUE_TEMPLATE/config.yml]: done to https://jetty.org/security.html

File: .github/ISSUE_TEMPLATE/release-template.md (hunk 3 above)
Comment (olamy) [reply .github/ISSUE_TEMPLATE/release-template.md]: to https://jetty.org/download.html

File: .github/ISSUE_TEMPLATE/issue-template.md (hunk 2 above)
Comment (olamy) [reply .github/ISSUE_TEMPLATE/issue-template.md]: done to https://jetty.org/security.html

**Comment by joakime** [comment]
The changes you have here have broken the `org.eclipse.jetty.tests.distribution.DistributionTests.testWebAppWithProxyAndJPMS` tests.

**Review by joakime — CHANGES_REQUESTED** [review]
Please address the new test failures (not flaky) of `org.eclipse.jetty.tests.distribution.DistributionTests.testWebAppWithProxyAndJPMS`

**Comment by olamy** [comment]
fixed

**Review by joakime — APPROVED** [review]

**Pull request merged**